pkgs/applications/networking/browsers/chromium-bin/default.nix

```diff
@@ -0,0 +1,8 @@
+{
+  callPackage,
+  stdenv,
+}:
+if stdenv.isDarwin
+then callPackage ./darwin.nix {}
+else throw "chromium-bin has not been packaged for linux. use chromium instead"
```

Comment:
This deviates from the common pattern of listing supported platforms explicitly in `meta.platforms`. I think that's better than going with a custom solution.

Reply:
I'm not sure what you mean. Could you point me to an example?

Reply:
Most any package expression will do as an example. It's just either not including a platform in `meta.platforms` or including it but adding it to `meta.broken` too.

Reply:
Oh, I see. The reason for doing it this way is because of the `extractDarwinApp`. Were a linux version to be introduced, it would require creating a new derivation, and IMO it would be better for that to be a separate file.
I imagine that would require a common meta attrset.

We can let the linux packager find out how to handle that. I'll just return the darwin package for now.

Reply:
Done in 5f5e9600e882b8540d4cc7edbdb7403692bde909